File: curso-python/ex080.py
lista = []

for c in range(0,5):
    n = int(input(f"digite o {c}°valor: "))

    if c == 0 or n > lista[-1]:
        lista.append(n)
        print("adicionado ao final da lista...")
    else:
        pos = 0
        while pos < len(lista):
            if n <= lista[pos]:
                lista.insert(pos, n)
                break
            pos += 1
print(f"os valores digitados em ordem foram {lista}")

File: javascript-exercicios/ex080.js
const prompt = require("prompt-sync")();

let lista = [];

for(let c=0;c<5;c++){
    let n = Number(prompt(`digite o ${c}°valor: `));

    if (c == 0 || n > lista[lista.length - 1]){
        lista.push(n)
        console.log("adicionado ao final da lista")
    }else{
        let pos = 0
        while (pos < lista.length){
            if (n <= lista[pos]){
                lista.splice(pos, 0, n)
                break
            }
            pos += 1
        }
    }
}

console.log(`os valores digitados em ordem foram ${lista}`)